main.go

```diff
@@ -122,6 +122,7 @@ Usage:
   ios setlocation [options] [--lat=<lat>] [--lon=<lon>]
   ios setlocationgpx [options] [--gpxfilepath=<gpxfilepath>]
   ios resetlocation [options]
+  ios simulatelocation [options] [--lat=<lat>] [--lon=<lon>]
```

Comment:
there are already these three commands to handle location simulation, lets re-use them
```
  ios setlocation [options] [--lat=<lat>] [--lon=<lon>]
  ios setlocationgpx [options] [--gpxfilepath=<gpxfilepath>]
  ios resetlocation [options]
```